ext/json/json_parser.y

```diff
@@ -212,18 +212,35 @@ value:
 
 %% /* Functions */
 
+static int php_json_parser_array_create_validate(php_json_parser *parser, zval *array)
+{
+	ZVAL_EMPTY_ARRAY(array);
+	return SUCCESS;
+}
+
```

Comment:
I would probably stick all these `_validate` methods together, instead of interspersing them among the "normal" parsing callbacks. It also makes the PR leaner and easier to read.

Reply:
Give me the time to analyse this so I can give you a good feedback

Reply:
@derickr 
So, you suggest to put them all together ... I mean, all validation hooks one next to the other? just relocate them in the file? Did I understood correctly?

Reply:
Done